omero_search_engine/api/v1/resources/resource_analyser.py

```diff
@@ -398,6 +395,18 @@ def get_key_values_return_contents(name, resource, csv):
     return jsonify(resource_keys)
 
 
+def adjuest_value(value):
```

Comment:
to be renamed to ``adjust_value``

Reply:
It has been changed to `adjust_value`, thank you.